index.dd

```diff
@@ -408,3 +408,34 @@ Macros:
     _=
 	FRONTPAGE_ORG_IMG=$(DIVC frontpage-orgs-img-wrapper vcontainer-box, <img class="vcontainer-element" src="$(ROOT_DIR)images/orgs-using-d/$0" />)
     FRONTPAGE_ORG=$(DIVC frontpage-orgs-cell dont-highlight-link, $(LINK2 $2, $(FRONTPAGE_ORG_IMG $3)) $(H3 $1) $(P $4))
+    C_A=$1:$2;
+    EXTRA_HEADERS=$(T style,
+        .frontpage-orgs h3 {
+           $(C_A margin, 0.4em)
```

Comment:
Ddoc!
It recognizes `.*:` as title (like `title:` or `params:`) and behaves weirdly.
I should probably add a comment about this.